src/index.js

```diff
@@ -23,7 +23,7 @@ Chart._adapters._date.override({
 	_create: function(time) {
 		return DateTime.fromMillis(time, this.options);
 	},
-
+	
```

Comment:
should remove this newly added whitespace

Reply:
done